GTG/gtk/browser/main_window.py

```diff
@@ -1315,6 +1315,26 @@ def on_dismiss_task(self, widget=None):
                 task.set_status(Task.STA_DISMISSED)
                 self.close_all_task_editors(uid)
 
+    def on_reopen_task(self, widget=None):
+        tasks_uid = [uid for uid in self.get_selected_tasks()
+                     if uid is not None]
+        if len(tasks_uid) == 0:
+            return
+        tasks = [self.req.get_task(uid) for uid in tasks_uid]
+        tasks_status = [task.get_status() for task in tasks]
+        for uid, task, status in zip(tasks_uid, tasks, tasks_status):
+            if status == Task.STA_DONE:
+                # Marking as undone
```

Comment:
This is already clear from the context.  I know you've copied from above, but if you're already here you can change this instance.